### PR TITLE
Fixes changeling power synaptic enhancement

### PR DESCRIPTION
**Pull request by ghost**
Now watch as we see the new meme strat of lings with arm blades attacking faster than star platinum.

### CODE REVIEW

**Comment by Carbonhell** [comment]
No, all the other powers use "chemical cost" variable, this one uses this hardcoded value and there's no reason at all for it to exist, try to use chemical cost and check how other powers work

**Comment by ghost** [comment]
I have a plan, gimme a few

**Comment by ghost** [comment]
Alright this should be good shit now

**Comment by ghost** [comment]
Wait no fuck it doesn't use up 30 chems.

File: code/game/gamemodes/changeling/powers/synaptic_enhancement.dm [hunk 1]
```diff
@@ -4,25 +4,23 @@
...
 /obj/effect/proc_holder/changeling/synaptic_enhancement/sting_action(mob/user)
 	var/datum/changeling/changeling=user.mind.changeling
 	if(!active)
-		if(chemical_cost <= 30) //Hardcoded chem cost
-			user << "<span class='warning'>We require at least 30 units to do that!</span>"
-			return
 		active = 1
-		changeling.chem_storage -= 30
 		user.next_move_modifier = 0.5
+		chemical_cost = 0
```
Comment (Carbonhell) [inline code/game/gamemodes/changeling/powers/synaptic_enhancement.dm]: what no you don't have to set it later on, remove it from here and  on the line 23

**Comment by Carbonhell** [comment]
code's fine, ready to merge?

**Comment by myontasm** [comment]
:+1: Sounds good to me.

**Comment by ghost** [comment]
It's ready to merge I think

**Pull request merged**